lib-multisrc/gigaviewer/src/eu/kanade/tachiyomi/multisrc/gigaviewer/GigaViewerDto.kt

```diff
@@ -24,3 +24,34 @@ data class GigaViewerPage(
     val type: String = "",
     val width: Int = 0,
 )
+
+@Serializable
+data class GigaViewerEpisodesDto(
+    val episodes: List<GigaViewerPaginationReadableProduct> = emptyList(),
+)
+
+@Serializable
+data class GigaViewerPaginationReadableProduct(
```

Comment:
Use a plain `class` instead of `data class` unless you're using `==`, `copy()` or `hashCode()`.